main.go

```diff
@@ -33,6 +37,22 @@ func main() {
 		logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
 	}
 
+	var sentryHook *logrus_sentry.SentryHook
```

Comment:
Can we move this initialization to a little  later (after we parse the crontab in `readCrontabAtPath`, and check that it succeeded). We don't need to set things up if the crontab doesn't even exist